### ISSUE TITLE
Context is missing on MutationOptions

### ISSUE THREAD

**Issue by helfer** [apollographql/apollo-client#2374]
**Intended outcome:**
`MutationOptions` should include `context` and pass it to the link.

**Actual outcome:**
Mutations do not accept context as an argument, only queries do.

**Version**
- apollo-client@next (cfd2561, 2.0-ish)

FWIW, I'm currently using the following link as a workaround at the top of my link stack:

```js
import { ApolloLink } from 'apollo-link';

export default new ApolloLink((operation, forward) => {
    if (operation.variables.myContext) {
        operation.setContext({
            ...operation.variables.myContext,
        });
        delete operation.variables.myContext;
    }
    return forward(operation);
});
```

**Comment by jbaxleyiii** [comment]
Fixed in the next rc!

Also 👋  ❤️

**Comment by gdi2290** [comment]
which release is this? @jbaxleyiii

**Comment by wittydeveloper** [comment]
Patched in 2.0.3 (https://github.com/apollographql/apollo-client/pull/2419#issuecomment-346376432)
